mr/src/main/java/org/elasticsearch/hadoop/rest/EsHadoopBulkException.java

```diff
@@ -0,0 +1,44 @@
+package org.elasticsearch.hadoop.rest;
+
+import org.elasticsearch.hadoop.EsHadoopException;
+
+public class EsHadoopBulkException extends EsHadoopException {
```

Comment:
I think this should be named something like `EsHadoopRemoteException` instead of having `Bulk` in the name. This error extractor is run for any failing request, which may not be a bulk related request in every case